docs/examples/basic-auth-api.md

```diff
@@ -0,0 +1,147 @@
+---
+title: "API using basic authentication"
+subtitle: "Example application that exposes an API behind basic authentication"
+type: 
+  - "pattern"
+platform:
+  - "awscdk"
+  - "tf-aws"
+  - "sim"
+language:
+  - "wing"
+githubURL: "https://github.com/winglang/examples.git && cd examples/examples/api-basic-auth"
```

Comment:
Perhaps have the directory as a separate (optional) field? I think package.json does it like this:
```
{
  "repository": {
    "type": "git",
    "url": "git+https://github.com/winglang/winglibs.git",
    "directory": "dynamodb"
  }
}
```

```suggestion
githubURL: "https://github.com/winglang/examples.git"
repoDirectory: "examples/api-basic-auth"
```

Reply:
Yeah cool, makes sense to me